src/librbd/image/RefreshRequest.cc

```diff
@@ -1320,6 +1320,13 @@ void RefreshRequest<I>::apply() {
       m_image_ctx.parent_md = m_parent_md;
       m_image_ctx.migration_info = {};
     }
+
+    librados::Rados rados(m_image_ctx.md_ctx);
+    int8_t require_osd_release;
+    int r = rados.get_min_compatible_osd(&require_osd_release);
+    if (r == 0 && require_osd_release >= CEPH_RELEASE_OCTOPUS) {
```

Comment:
Need to rebase and tweak

Reply:
@dillaman Not sure what you mean here. I rebased but there were no any confilicts, there seems nothing to tweak.

Reply:
Never mind, I was thinking it might have been broken by PR 27855, but it wasn't.